.travis.yml

```diff
@@ -97,7 +97,7 @@ before_script:
 script:
   - sudo iptables -I OUTPUT 1 -p udp --dport 10086 -j REJECT
   - cd luajit2/
-  - make -j$JOBS CCDEBUG=-g Q= PREFIX=$LUAJIT_PREFIX CC=$CC XCFLAGS='-DLUA_USE_APICHECK -DLUA_USE_ASSERT -msse4.2' > build.log 2>&1 || (cat build.log && exit 1)
+  - make -j$JOBS CCDEBUG=-g Q= PREFIX=$LUAJIT_PREFIX CC=$CC XCFLAGS='-DNGX_HTTP_LUA_SHDICT_HAVE_RWLOCK -DLUA_USE_APICHECK -DLUA_USE_ASSERT -msse4.2' > build.log 2>&1 || (cat build.log && exit 1)
```

Comment:
i suspect this may not be desirable in a final commit; this just demonstrates the validity of the locking mechanism in unit tests.